extensions/azure/fcc-node-directory-cosmos/build.gradle.kts

```diff
@@ -36,12 +36,4 @@ publishing {
             from(components["java"])
         }
     }
-}
-publishing {
-    publications {
-        create<MavenPublication>("in-memory.asset-resolver") {
-            artifactId = "in-memory.asset-resolver"
-            from(components["java"])
-        }
-    }
-}
+}
```

Comment:
can you change this to `fcc-node-directory-cosmos`? we'll need that maven artifact.

Reply:
is already like that: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/0bef2f8aecb01c9a2d9b33d46b7bbb084fc5dad8/extensions/azure/fcc-node-directory-cosmos/build.gradle.kts#L34

Reply:
ok, sorry, i must have gotten an out dated version. will approve then.